mobile-browser-based-version/src/helpers/training_script/training-script.js

```diff
@@ -139,3 +140,121 @@ export async function trainingDistributed(
       console.log('Training finished', info.history);
     });
 }
+
+/**
+ * This function is called when we decided to train using personalized. It creates a local and a global model.
```

Comment:
mention the kind of personalization.
also in the UI, it's maybe not very obvious yet if the user only selects interoperable, or personalization?

Reply:
For now the only personalization is the interoperability, but yeah you're right i'll make it clearer

Reply:
For now the UI looks like this and only mentions Interoperability but in the future more options can be added. 
<img width="351" alt="Capture d’écran 2021-11-03 à 11 52 34" src="https://user-images.githubusercontent.com/36693361/140047937-3c71eb65-943d-4cf2-866d-a2610f079f00.png">